### PR TITLE
[Unity] View Functionality Added

### PR DESCRIPTION
**Pull request by AliRKat**

### CODE REVIEW

**Comment by AliRKat** [comment]
[[Unity, Doc] Add new view methods remove deprecated ones](https://trello.com/c/s1HwIBFY/1818-unity-doc-add-new-view-methods-remove-deprecated-ones)

File: unity/next.md [hunk 1]
```diff
@@ -415,39 +415,178 @@ Countly.Instance.Events.CancelEvent(eventName);</code></pre>
   If consent was removed and then given, and automatic sessions were enabled, a
   session will automatically be started.<span></span>
 </p>
-<h1 id="h_01HABTZ315E40XEHJHRMQMW0J6">View Tracking</h1>
```
Comment (turtledreams) [inline unity/next.md:418]: these titles could have links pointing at them. These links are would use the id of the title. so tittle ids should not change

**Review by turtledreams — CHANGES_REQUESTED** [review]

File: unity/next.md [hunk 2]
```diff
@@ -415,39 +415,178 @@ Countly.Instance.Events.CancelEvent(eventName);</code></pre>
...
-<h2 id="h_01HABTZ31504DKRD8DBNE1W400">Manual View Recording</h2>
+<h1 id="h_01HAVQDM5T4KYTSDG1BQ41WDM1">View Tracking</h1>
 <p>
-  The Countly Unity SDK supports manual view (screen) tracking. With this feature,
-  you can report what views a user did and for how long. Thus, whenever there is
-  a screen switch in your app, you can report it to the Countly server by using
-  the following method:
+  In the SDK, all view-related functionality can be browsed from the returned interface
```
Comment (arifBurakDemiray) [inline unity/next.md:420]: please also add user guide documentation for it
https://support.count.ly/hc/en-us/articles/4444616740249-Flows

**Review by arifBurakDemiray — APPROVED** [review]

File: unity/next.md [hunk 3]
```diff
@@ -416,38 +416,181 @@ Countly.Instance.Events.CancelEvent(eventName);</code></pre>
...
+viewSegmentation.Add("HasStreamingSubscription", true);
+viewSegmentation.Add("ScreenSize", 55.0f);
+viewSegmentation.Add("UserID", 5566778899L);
+  
+Countly.Instance.Views.UpdateGlobalViewSegmentation(viewSegmentation);</code></pre>
+<p>
+  To review the resulting data from view tracking, open the dashboard and go to
+  <code>Analytics &gt; Views</code>. For more information on using view tracking
+  data to its fullest potential, click
+  <a href="http://resources.count.ly/docs/view-analytics" target="_blank" rel="noopener noreferrer">here</a>.
```
Comment (turtledreams) [inline unity/next.md:589]: isnt there something more up to date?
also maybe it would be better to move this paragraph to the top of the feature h1

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by peterBrxwn — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**